src/components/GameCompleteDialog/index.scss

```diff
@@ -0,0 +1,65 @@
+.game-complete-dialog__podium {
+    display: grid;
+    grid-template-areas: 
+        '...... first .....'
+        'second first .....'
+        'second first third'
+        '...... first .....';
+    grid-template-columns: repeat(3, min-content);
+    grid-template-rows: repeat(3, min-content) 10px;
+
+    .podium {
```

Comment:
why didn't you use the inheritance here?

```scss

.podium {

  &--first {}
```

etc.

Reply:
a0184c3cf3a272542dde6f7baa8ba5700c3aa291